src/components/StatusLabel/StatusLabel.tsx

```diff
@@ -29,7 +29,8 @@ export const useStyles = makeStyles(
       textContainer: {
         marginLeft: theme.spacing(1),
         display: "flex",
-        flexDirection: "column"
+        flexDirection: "column",
+        width: "100%"
```

Comment:
I'm not sure if this wouldn't break anything. Mind checking how does this component renders now in other places?

Reply:
I went through majority if not all the dashboard places and it looks like nothing has changed.